static/js/components/EducationDisplay.js

```diff
@@ -117,7 +119,7 @@ export default class EducationDisplay extends ProfileFormFields {
         <Card shadow={1} className="profile-form" id="education-card">
           <Grid className="profile-form-grid">
             <Cell col={12} className="profile-form-row profile-card-header">
-              <span>
+              <span className="profile-form-card-header">
```

Comment:
why not just change this to `.title`? then you can style it by doing:

```css
.profile-card-header {
  .title {
    font-weight: 600;
  }
}
```

or any other styles you want to add.